lib/rspec/matchers/built_in/be.rb

```diff
@@ -75,7 +75,7 @@ module BeHelpers
       private
 
         def args_to_s
-          @args.empty? ? "" : parenthesize(inspected_args.join(', '))
+          @args.empty? ? '' : parenthesize(inspected_args.join(', '))
```

Comment:
Ah that make sense, I was simply running `rubocop lib`. Thank you for pointing that out.